[demo_decode.py] | (Galaxy) Stress | galaxy-stress-test / Llama Galaxy Long Stress Test [wh_galaxy_perf]

## CI Failure Report

<!-- AUTO-TRIAGE-SUMMARY-START -->
**Summary:** [CI] Llama Galaxy Long Stress Test fails with OSError: Read-only file system on /mnt/MLPerf model path
<!-- AUTO-TRIAGE-SUMMARY-END -->

**Workflow:** (Galaxy) Stress
**Job:** galaxy-stress-test / Llama Galaxy Long Stress Test [wh_galaxy_perf]

### Regression timeline

- **Last passing commit:** N/A
- **First failing commit:** `565320529689` — 2026-06-06T04:27:11Z — [job](https://github.com/tenstorrent/tt-metal/actions/runs/27052543210/job/79852356866)
- **Most recent failing commit:** `f1e8f9b60d09` — 2026-06-13T04:34:52Z — [job](https://github.com/tenstorrent/tt-metal/actions/runs/27456686646/job/81162870469)

### Failing job URLs (last 2 runs)

- https://github.com/tenstorrent/tt-metal/actions/runs/27456686646/job/81162870469
- https://github.com/tenstorrent/tt-metal/actions/runs/27052543210/job/79852356866

### Failing test path(s)

- `models/demos/llama3_70b_galaxy/demo/demo_decode.py::test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test]`

### Error excerpt (terminal failure from logs)

**Run 1 (2026-06-13):**
```
##[error]test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test]

OSError: [Errno 30] Read-only file system: '/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG'

/usr/local/share/uv/cpython-3.10.19-linux-x86_64-gnu/lib/python3.10/os.py:225: OSError

FAILED models/demos/llama3_70b_galaxy/demo/demo_decode.py::test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test] - OSError: [Errno 30] Read-only file system: '/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG'
1 failed, 5 deselected, 4 warnings in 14.39s
```

**Run 2 (2026-06-06):**
```
##[error]test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test]

OSError: [Errno 30] Read-only file system: '/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG'

FAILED models/demos/llama3_70b_galaxy/demo/demo_decode.py::test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test] - OSError: [Errno 30] Read-only file system: '/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG'
```

### Reproduction steps

Run the Llama Galaxy Long Stress Test locally on a `wh_galaxy_perf` runner:

```bash
pytest models/demos/llama3_70b_galaxy/demo/demo_decode.py::test_llama_demo \
  -k 'wormhole_b0-device_params0-mesh_device0-performance-stress-test'
```

Ensure that `/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG` is writable on the runner host. The test attempts to create a directory at that path (via `os.makedirs`) but the mount is read-only.

### Notes

- The failure occurs after the model runs inference successfully (Real-time profiler completes 100 samples on all devices), then at the "Start profiler" / "Reading inputs" phase where the test tries to create a directory under `/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG`.
- The NFS/CIFS/fuse mount at `/mnt/MLPerf` appears to have become read-only on the runner, preventing directory creation.
- This is an infrastructure issue — the model weights mount is read-only, not a code regression per se, but it is consistently reproducible.
- **Determinism check:**
  - Run 1 (commit `f1e8f9b60d09`, 2026-06-13): `FAILED models/demos/llama3_70b_galaxy/demo/demo_decode.py::test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test] - OSError: [Errno 30] Read-only file system: '/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG'`
  - Run 2 (commit `565320529689`, 2026-06-06): `FAILED models/demos/llama3_70b_galaxy/demo/demo_decode.py::test_llama_demo[wormhole_b0-device_params0-mesh_device0-performance-stress-test] - OSError: [Errno 30] Read-only file system: '/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/TG'`
  - Both runs: identical error class (`OSError [Errno 30]`), identical path, identical test ID.

<!-- AUTO-TRIAGE-METADATA-START -->
`Auto-triage-workflow: (Galaxy) Stress`
`Auto-triage-job-name: galaxy-stress-test / Llama Galaxy Long Stress Test [wh_galaxy_perf]`
<!-- AUTO-TRIAGE-METADATA-END -->

<!-- PIPELINE-REPORT-START -->
## Pipeline report
Analysis report: not found
Slack notification: not found
<!-- PIPELINE-REPORT-END -->

<!-- AUTO-TRIAGE-INFO-START -->
## About this issue (automated CI auto-triage)
This issue was opened and assigned by an automated AI system. It posts through a bot token, so notifications may look like they come from a person, but they are automated.

The owner is selected from recent commit history and job-ownership data. It reflects who is *likely able to fix or route* this, not necessarily who caused it, so it can be imperfect.

**If this was assigned to the wrong person, you can:**
- Comment `REASSIGN OWNER: <why this is outside your area / which team or test-area it belongs to>` and the system will automatically re-route it to a better-suited owner.
- Unassign yourself (this stops notifications to you).
- Close the issue if it is not a real or relevant failure.

The assigned owner is reminded about every 3 days until the issue is resolved. Full guide: https://tenstorrent.atlassian.net/wiki/spaces/MI6/pages/2545221692/Auto-Triage+Info
<!-- AUTO-TRIAGE-INFO-END -->

--- a/.github/workflows/galaxy-stress-tests-impl.yaml
+++ b/.github/workflows/galaxy-stress-tests-impl.yaml
@@ -112,7 +112,6 @@ jobs:
         LD_LIBRARY_PATH: /work/build/lib
         LOGURU_LEVEL: INFO
         ARCH_NAME: wormhole_b0
-        LLAMA_DIR: /mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/
       volumes:
         - ${{ github.workspace }}/docker-job:/work
         - /dev/hugepages-1G:/dev/hugepages-1G

--- a/.github/workflows/galaxy-stress-tests.yaml
+++ b/.github/workflows/galaxy-stress-tests.yaml
@@ -36,9 +36,7 @@ on:
         default: "all"
         options:
           - all
-          - galaxy-llama-nd-hang-stress
           - galaxy-fabric-stability-stress
-          - galaxy-llama-long-stress
       extra-tag:
         required: false
         type: string

--- a/tests/pipeline_reorg/galaxy_stress_tests.yaml
+++ b/tests/pipeline_reorg/galaxy_stress_tests.yaml
@@ -10,19 +10,6 @@
 
 # Max timeouts: .github/time_budget.yaml (workflow key: stress)
 
-- id: galaxy-llama-nd-hang-stress
-  name: Llama Galaxy ND hang stress (loop)
-  team: models
-  owner_id: U053W15B6JF # Djordje Ivanovic
-  cmd: |
-    for i in {1..5}; do
-      echo "🔁 Run #$i"
-      timeout --preserve-status 1200 pytest models/demos/llama3_70b_galaxy/demo/demo_decode.py -k "nd-hang-test"
-    done
-  skus:
-    wh_galaxy_perf:
-      timeout: 360
-
 - id: galaxy-fabric-stability-stress
   name: Galaxy Fabric Stability Tests
   team: scaleout
@@ -32,13 +19,3 @@
   skus:
     wh_galaxy_perf:
       timeout: 100
-
-- id: galaxy-llama-long-stress
-  name: Llama Galaxy Long Stress Test
-  team: models
-  owner_id: U053W15B6JF # Djordje Ivanovic
-  cmd: |
-    LLAMA_DIR=/mnt/MLPerf/tt_dnn-models/llama/Llama3.3-70B-Instruct/ FAKE_DEVICE=TG pytest models/demos/llama3_70b_galaxy/demo/demo_decode.py -k "stress-test and not mini-stress-test"
-  skus:
-    wh_galaxy_perf:
-      timeout: 140

